sdks/python/apache_beam/examples/inference/tfx_bsl/build_tensorflow_model.py

```diff
@@ -0,0 +1,71 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import tensorflow as tf
+
+
+class TFModelWrapperWithSignature(tf.keras.Model):
```

Comment:
The idea here is to provide users a simple wrapper in which the signature is predefined for images and deserializing the `Example` Proto is also done along with deserializing `tensor` as well. 

This can be used as an example for the users on how to build a tensorflow model with modified signature. 

Also, the parameters in the constructor can be used for testing only purpose.

Reply:
+1 on adding a class header function here, I think making this step clear could be one of the biggest value adds of this example

Reply:
I will add it once the postcommit passes